main/src/library/Foldable.flix

```diff
@@ -49,6 +49,11 @@ pub trait Foldable[t : Type -> Type] {
     pub def length(t: t[a]): Int32 \ Foldable.Aef[t] =
         Foldable.foldLeft((acc, _) -> acc + 1, 0, t)
 
+    ///
+    /// Returns the length of `t`, i.e. the number of times it can be folded over.
+    ///
+    pub def size(t: t[a]): Int32 \ Foldable.Aef[t] = Foldable.length(t)
```

Comment:
It might potentially be an issue to have both on the trait since if you only `redef` size, you get asymmetrical behaviour on what is supposed to be an alias